draft-kazuho-httpbis-priority.md

```diff
@@ -139,6 +131,39 @@ The following example shows a request for a CSS file with the urgency set to
 priority = urgency=blocking
 ~~~
 
+The definition of the urgencies and their expected use-case are described below.
+
+### blocking
+
+The `blocking` urgency indicates that the response prevents other responses from
+being used.
+
+For example, a stylesheet being referred from the HEAD section of an HTML
+document blocks a web browser from rendering the HTML.  In such case, the
+stylesheet is given the `blocking` urgency.
+
+### document
+
+The `document` urgency indicates that the response contains the document that is
+being processed.  This urgency is also used for responses that deserve the same
+precedence as the contents of the document.
+
+For example, when a user using a web browser navigates to a new HTML document,
+the request for that HTML is given the `document` urgency.  When that HTML
+document uses a custom font, HTTP request for that custom font SHOULD also be
+given the `document` urgency, because it is generally assumed that the usability
+of the HTML document partially depends on the availability of the font, even
+though the font does not block the use of the document as a whole.
+
+### non-blocking
+
+The `non-blocking` urgency indicates that the response does not prevent the
+client from using the document even though the response is being used or
+referred to by the document.
+
+For example, inline images typically improves the usability of an HTML document.
```

Comment:
either:

a) `inlining images typically improves`
b) `inline image typically improve`

Reply:
Went for plural in d76ba24.